### PR TITLE
Fix #21042: SpriteData intialised incorrectly

### LINKED ISSUE #21042: Guests have rendering issues in queue lines
### Operating System

Windows 11, 64-bits

### OpenRCT2 build

OpenRCT2, v0.4.6-174-g004592b (004592b on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

It will show any queue lines whit guests glitching

### Steps to reproduce

1. Open any scenario (my example is in Diamond heights whit the Snake River Falls Log Flume's queue line)
2. Check any non-covered queue line
3. Guests will partially dissapear

### Attachments

https://github.com/OpenRCT2/OpenRCT2/assets/83843734/d7e4d69d-038d-41fa-a27f-8cc5ded50fcb

### PR DESCRIPTION
**Pull request by mrmbernardi**
Fixes #21042

It took a lot of debugging to find this one

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Lovely find!

Looking at `git blame`, this issue was introduced before the last release, so I think a changelog entry is in order.

**Review by ZehMatt — APPROVED** [review]
Good catch.

**Comment by mrmbernardi** [comment]
I've added a changelog entry

File: distribution/changelog.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@
 - Fix: [#20807] Tertiary colour not copied with small scenery.
 - Fix: [#20964] Crash when player connects to server with a group assigned that no longer exists.
 - Fix: [#20995] TTF fonts don’t support hinting, outlines, or insets with OpenGL.
+- Fix: [#21042] Peeps don't render properly in S4 parks.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:32]:
```suggestion
- Fix: [#21042] Peeps don’t render properly in S4 parks.
```

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**